src/contrib/cluster/Akka.Cluster.Sharding.Tests.MultiNode/ClusterShardingSpec.cs

```diff
@@ -156,14 +191,15 @@ private Stop()
         public static readonly ShardResolver ExtractShardId = message =>
         {
             if (message is EntityEnvelope)
-                return (((EntityEnvelope)message).Id % ShardsCount).ToString();
+                return (((EntityEnvelope)message).Id % NumberOfShards).ToString();
```

Comment:
i didn't want to use any c# 7 in existing code to keep the # of changes minimal.
Was planning to update this in another pr later on. But if you want, I can change everything to c# 7, but this will increase changes significantly. Shall I change it then in this PR or later?